src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -42,6 +43,12 @@
         # In snowflake, a double single quote resolves as a single quote in the string.
         # https://docs.snowflake.com/en/sql-reference/data-types-text.html#single-quoted-string-constants
         RegexLexer("single_quote", r"'([^'\\]|\\.|'')*'", CodeSegment),
+        RegexLexer(
+            "inline_comment",
+            r"(--|#|//)[^\n]*",
+            CommentSegment,
+            segment_kwargs={"trim_start": ("--", "#")},
```

Comment:
Should we not also add the `//` here? Not that we usually do anything with the comments, so not sure this DOES anything, but for consistency we should.

Reply:
Yup - just noticed the same thing. Fixed 👍